lib/tools/checkParamObject.js

```diff
@@ -7,9 +7,13 @@ module.exports = function (userParam, defaultParam, auditKey) {
   let defaultProp
   let extraOrMissingkeysBool = false
 
+  // params to skip
+  let skipExtra = ['logging', 'multipart', 'bodyParser']
+  let skipMissing = ['multipart', 'bodyParser']
```

Comment:
let's declare these in config.json

i'm thinking something like this: if we add a new param like `"_roosevelt-exempt": "true"` to things in config.json that can have freeform params, the auditor will simply ignore that param and not audit it

Reply:
In order to smoothly transition to that you'll have to ensure that the auditor ignores that key, keeping in mind that the auditor and param setter share resources.